backend/tests/test_participant_details.py

```diff
@@ -22,21 +22,54 @@ def test_get_delete_participant_when_logged_in(
     else:
         assert response["message"] == "Participant deleted successfully."
 
+@pytest.mark.parametrize("method", ["put"])
```

Comment:
You could remove that line  - parametrize is useful when you have more arguments - there is one only, and you don't use it in test.